man/systemctl.xml

```diff
@@ -1245,6 +1245,28 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
         </varlistentry>
 
         <varlistentry>
+          <term><command>revert <replaceable>NAME</replaceable>...</command></term>
```

Comment:
We already have "systemctl reset-failed", and I'd really would like to avoid the name "reset" for anything else because of that. Also we use the term "factory reset" already for stateless systems where you flush out /etc...

Before git, "svn revert" was about returning your worktree to the upstream version. I'd thus claim that "revert" already has many different meanings, and this one certainly fits in among the existing meanings...